MODULE.bazel

```diff
@@ -10,3 +10,33 @@ bazel_dep(name = "aspect_bazel_lib", version = "1.20.0")
 bazel_dep(name = "bazel_skylib", version = "1.3.0")
 bazel_dep(name = "rules_nodejs", version = "5.5.3")
 bazel_dep(name = "platforms", version = "0.0.4")
+
+node = use_extension("@rules_nodejs//nodejs:extensions.bzl", "node")
+
+node.toolchain(
+    name = "nodejs",
+)
+
+use_repo(node, "nodejs_host")
+use_repo(node, "nodejs_toolchains")
+
+use_repo(node, "nodejs_linux_amd64")
+use_repo(node, "nodejs_darwin_arm64")
+use_repo(node, "nodejs_darwin_amd64")
+use_repo(node, "nodejs_linux_arm64")
+use_repo(node, "nodejs_linux_s390x")
+use_repo(node, "nodejs_linux_ppc64le")
+use_repo(node, "nodejs_windows_amd64")
```

Comment:
Unfortunate, but seems to be required. See my comment about not being able to use `@nodejs_host` to access the binary below.

Reply:
hmmm. that is unfortunate. guessing `use_repo(node, "nodejs_host")` is not needed in above anymore?

a user would only have to specify the `nodejs_<platform>` repository for the hosts they need I'm guessing?

Reply:
Users shouldn't have to declare this at all. It just had to be in our own MODULE>bazel. Right I don't think `@nodejs_host` is needed anymore.

Reply:
Removed.